google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -95,6 +95,186 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
   private final Long retentionEffectiveTime;
   private final Boolean retentionPolicyIsLocked;
   private final Long retentionPeriod;
+  private final IamConfiguration iamConfiguration;
+
+  /**
+   * A Bucket's IAM Configuration. Allows specification of authorization policies via IAM instead of legacy ACL.
```

Comment:
I'd leave this simplified to: `The bucket's IAM configuration.` per the discovery document.
https://www.googleapis.com/discovery/v1/apis/storage/v1/rest